src/els_diagnostics_utils.erl

```diff
@@ -24,10 +24,16 @@ dependencies(Uri) ->
 dependencies([], Acc) ->
   Acc;
 dependencies([Uri|Uris], Acc) ->
-  {ok, [Document]} = els_dt_document:lookup(Uri),
-  Deps = els_dt_document:pois(Document, [behaviour, parse_transform]),
-  IncludedUris = included_uris(Document),
-  dependencies(Uris ++ IncludedUris, Acc ++ [Id || #{id := Id} <- Deps]).
+  case els_dt_document:lookup(Uri) of
+    {ok, [Document]} ->
+      els_dt_document:lookup(Uri),
+      Deps = els_dt_document:pois(Document, [behaviour, parse_transform]),
+      IncludedUris = included_uris(Document),
+      dependencies(Uris ++ IncludedUris, Acc ++ [Id || #{id := Id} <- Deps]);
+    Error ->
+      lager:info("els_diagnostics_utils:dependencies [Error=~p]", [Error]), %%AZ
```

Comment:
Nit: lager already takes care of including pid, module, function and line number in the log messages.

Reply:
Also, let's try not use initials in comments or todo. They are hard to understand for other contributors.

Reply:
My approach is to put initials in for my debug comments, I neglected to remove it before pushing. Agree they should not be there in the repo